lib/Jsrt/JsrtDebugUtils.cpp

```diff
@@ -467,7 +473,12 @@ const char16 * JsrtDebugUtils::GetClassName(Js::TypeId typeId)
     case Js::TypeIds_Promise:           return _u("Promise");
     case Js::TypeIds_GlobalObject:      return _u("Object");
     case Js::TypeIds_SpreadArgument:    return _u("Spread");
-
+#ifdef ENABLE_WASM
+    case Js::TypeIds_WebAssemblyModule:  return _u("WebAssembly.Module");
```

Comment:
I'm not entirely sure what the other browsers do. 
I looked at https://github.com/WebAssembly/design/blob/master/JS.md#webassemblymoduleprototype--tostringtag--property
To determine which string to use, but maybe it's not the right one.